p2p/src/peer_manager/peerdb/mod.rs

```diff
@@ -71,9 +76,7 @@ impl<T: NetworkingService> From<&PeerContext<T>> for ConnectedPeer {
 }
 
 // TODO: Store available addresses in a binary heap (sorting by their availability).
```

Comment:
I'm not sure why you're using the word "heap" here... a heap is a specific data structure. It does support some kind of order, but usually it solves niche problems. A BTreeMap is more than enough to provide some order.

Reply:
These words were already there. I am also not sure why use "heap", but I expect this part to change so I have not spent much time looking into it.